front/src/Api/iframe/Sound.ts

```diff
@@ -0,0 +1,59 @@
+import type { LoadSoundEvent } from '../Events/LoadSoundEvent';
+import type { PlaySoundEvent } from '../Events/PlaySoundEvent';
+import type { StopSoundEvent } from '../Events/StopSoundEvent';
+import { IframeApiContribution, sendToWorkadventure } from './IframeApiContribution';
+import SoundConfig = Phaser.Types.Sound.SoundConfig;
+
+export class Sound {
+    constructor(private url: string) {
+        sendToWorkadventure({
+            "type": 'loadSound',
+            "data": {
+                url: this.url,
+            } as LoadSoundEvent
+
+        });
+    }
+
+    public play(config: SoundConfig) {
```

Comment:
should this be removed ?

Reply:
Mmmm... no, it is the only way to play a sound right?

```
WA.sound.loadSound('url.mp3).play();
```

Reply:
https://github.com/thecodingmachine/workadventure/commit/fac6574cb5adfffbff8a94f236d1b0cf2cbaad40#diff-b7fd0a9e00605c085d7277dee486fda91141c2a4fa04989151d18691a7cccfc9

well in this commit a playSound method got removed

Reply:
oh but its seems like it was a different method nevermind

Reply:
Yes, it was kind of a shortcut to do `WA.sound.loadSound('url.mp3).play();`
And I'd rather avoid shortcuts and put those in a utility library instead.